hw_3/exceptions/NotFoundError.js

```diff
@@ -0,0 +1,6 @@
+export class NotFoundError extends Error {
+    constructor(message) {
+        super();
```

Comment:
I guess you can just pass the message variable to the parent constructor
```javascript
 constructor(message) {
        super(message);
  }      
```

Reply:
done